src/bench/logging.cpp

```diff
@@ -42,12 +42,12 @@ static void LogPrintLevelWithoutThreadNames(benchmark::Bench& bench)
 
 static void LogPrintWithCategory(benchmark::Bench& bench)
```

Comment:
nit: fn name (+ below) should be updated to `LogDebugWithCategory`

```suggestion
static void LogDebugWithCategory(benchmark::Bench& bench)
```